BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -6797,6 +7265,13 @@ private static void InitializeStyles()
             }
         }
     }
-
     #endregion UI //borrowing code from ModularMissile GUI
+    internal class RaycastHitComparer : IComparer<RaycastHit>
```

Comment:
This is the third identical `RaycastHitComparer` class in the codebase (ExplosionFX.cs, MissileBase.cs, and ModuleWeapon.cs). Maybe we should move it to somewhere in Utils, e.g., in OtherUtils.cs since that has a couple of LoS functions already?